impl/trafficgen/main.go

```diff
@@ -0,0 +1,148 @@
+package main
```

Comment:
is this intended to be an integration test?

may be worth renaming the package as such

Reply:
yeah probably. I'll rename the package.